lib/kubernetes-deploy/kubernetes_resource/pod.rb

```diff
@@ -33,7 +36,7 @@ def sync(mediator)
         @containers.each(&:reset_status)
       end
 
-      display_logs(mediator) if unmanaged? && deploy_succeeded?
+      @pod_logger.log(mediator: mediator, pod: self) if unmanaged?
```

Comment:
I'm passing the entire pod, since the PodLogger calls `fetch_logs` with different args depending on if we're streaming or not.